PowerPointLabs/PowerPointLabs/Models/PowerPointMagnifyingSlide.cs

```diff
@@ -146,9 +145,33 @@ private void AddZoomSlideCroppedPicture(float magnifyRatio = 1.0f)
             cropShape.Select();
             PowerPoint.Selection sel = Globals.ThisAddIn.Application.ActiveWindow.Selection;
             PowerPoint.Shape croppedShape = CropToShape.Crop(zoomSlideCopy, sel, magnifyRatio: magnifyRatio);
-            croppedShape.Cut();
+            try
+            {
+                string tempFilePath = FileDir.GetTemporaryPngFilePath();
+                Utils.GraphicsUtil.ExportShape(croppedShape, tempFilePath);
+                zoomSlideCroppedShapes = _slide.Shapes.AddPicture2(tempFilePath,
+                    Office.MsoTriState.msoFalse,
+                    Office.MsoTriState.msoTrue,
+                    0,
+                    0);

```

Comment:
Entering a single 0 on a new line looks a little out of place to me. @leeyh20 Is this a common coding practice for C#? I'm not sure.